commerce-theme-minium/commerce-theme-minium-impl/src/main/resources/META-INF/resources/render/view.jsp

```diff
@@ -20,14 +20,24 @@
 CPContentHelper cpContentHelper = (CPContentHelper)request.getAttribute(CPContentWebKeys.CP_CONTENT_HELPER);
 
 CPCatalogEntry cpCatalogEntry = cpContentHelper.getCPCatalogEntry(request);
+
 CPSku cpSku = cpContentHelper.getDefaultCPSku(cpCatalogEntry);
 
 long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 
-String addToCartId = PortalUtil.generateRandomKey(request, "add-to-cart");
+String addToCartId = PortalUtil.generateRandomKey(request, "addToCart");
+String ddmFormContainerId = PortalUtil.generateRandomKey(request, "formContainer");
 String galleryId = PortalUtil.generateRandomKey(request, "gallery");
 %>
 
+<script>
```

Comment:
@ethanbustad I don't like this

Reply:
Don't like the `<script>` tag here? Or defining global variables at all?

Reply:
defining global variables

Reply:
This global variables are a fix for the navigation. they are used in the `aui:script` at line 99.
The `script` tag is needed for timing reasons, we need these variables not cached and executed immediately.

So if we want to remove global variables we need to find other ways to solve this problem